env.md

```diff
@@ -0,0 +1,3 @@
+# Environment variable test by {{process.env.ORGANIZATION_NAME}}
```

Comment:
🐞 

This part demonstrates a bug. In the body of page (h1 and body paragraph), the environment variable is replaced. (Also in the `siteConfig.yaml`.) However, the markdown page uses the first h1 as the title tag if there is none specified in the front matter. In the case of the title tag, it is not replaced. For clarification, the part that is generated from the `siteConfig.yaml` is replaced. The other part is not replaced, and it says this within my development environment.

> Environment variable test by {{process.env.ORGANIZATION_NAME}} · Redocly Training by Example Corp.

Reply:
The same issue if we try to use it within the front matter itself:

```
---
title: Test {{process.env.ORGANIZATION_NAME}}
---
```

Corresponding title:

> Test {{process.env.ORGANIZATION_NAME}} · Redocly Training by Example Corp.